### PR TITLE
Generate right URL in the OGC capabilities 

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/backend_qgis.rst [hunk 1]
```diff
@@ -134,3 +134,19 @@ like that:
    user=${PGUSER}
    password=${PGPASSWORD}
    port=${PGPORT}
+
+OGC server
+==========
+
+To use the QGIS server in authenticated mode throw the mapserv proxy, it's required to be in docker mode,
```
Comment (pvalsecc) [inline doc/integrator/backend_qgis.rst]: What do you mean by "throw" in that context?

File: geoportal/c2cgeoportal_geoportal/views/entry.py [hunk 2]
```diff
@@ -644,9 +644,9 @@ def _fill_internal_wms(self, l, layer, wms, wms_layers, errors):
             resolutions = self._get_layer_resolution_hint(wms_layer_obj)
             if resolutions[0] <= resolutions[1]:
                 if "minResolutionHint" not in l:
-                    l["minResolutionHint"] = float("{0:0.2f}".format(resolutions[0]))
+                    l["minResolutionHint"] = float("{:0.2f}".format(resolutions[0]))
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/views/entry.py]: What do you want to achieve, here? I don't understand the back and forth with string...

File: geoportal/c2cgeoportal_geoportal/views/ogcproxy.py [hunk 3]
```diff
@@ -92,12 +92,10 @@ def _get_ogc_server(self):
             self.external_ogc_server if self.external else self.default_ogc_server
         )
 
-    def _get_wms_url(self):
+    def _get_wms_url(self,):
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/views/ogcproxy.py]: Why adding the `,`?

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/views/ogcproxy.py]: Oups...

File: doc/integrator/backend_qgis.rst (hunk 1 above)
Comment (sbrunner) [reply doc/integrator/backend_qgis.rst]: Fixed :-)

File: geoportal/c2cgeoportal_geoportal/views/entry.py (hunk 2 above)
Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/views/entry.py]: Non functional change, just removing unneeded position

**Pull request merged**